Add development history (#30)
precice/fluent-adapter: README.md

Our [wiki](https://github.com/precice/fluent-adapter/wiki) will help you start.

## Development history

> The initial version of this adapter was developed by Bernhard Gatzhammer during the work on his thesis [1].

Reviewer: ```suggestion
The initial version of this adapter was developed by Bernhard Gatzhammer as part of his doctoral dissertation [1].
```
